src/unity/schemas/IdentityUser.yml

```diff
@@ -0,0 +1,37 @@
+properties:
+  id:
+    type: string
+    description: user's id in quartz
+  idpeId:
+    type: string
+    description: user's id in idpe
+  email:
+    type: string
+    description: user's email
+  firstName:
+    type: string
+    description: user's first name
+  lastName:
+    type: string
+    description: user's last name
```

Comment:
What are we going to return for users that don't have first/last name? `null` or empty string? If `null`, then these need to be nullable. If empty string, then perhaps we should mention that here so that the UI team knows that they need to address that case.

Reply:
We went ahead and updated these to be nullable. We did reach out to the UI team to see how they would like to handle this and are pending a response.